ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -713,7 +713,11 @@ class ServicesHomeScreen extends React.Component<Props, State> {
         >
           <ServicesTab
             sections={nationalTabSections}
-            isRefreshing={isLoadingServices || pot.isLoading(potUserMetadata)}
+            isRefreshing={
```

Comment:
Since this condition is computed multiple times inside rendere function, could you create a const to be used in all these cases?

```typescript
const isRefreshing={
              isLoadingServices ||
              pot.isLoading(potUserMetadata) ||
              pot.isUpdating(potUserMetadata)
            }
```